### DISCUSSION TITLE
HELPLESS WITH THESE THREE ERRORS - PLEASE GUIDE TO RECTIFY?

### DISCUSSION THREAD

**Discussion by tmprabubiz** [lllyasviel/stable-diffusion-webui-forge#1239] [Q&A]
WINDOWS 10 PRO PC / i9 PROCESSOR / GPU: RTX 2060 SUPER 8GB / CPU: 64GB / USING SYMLINKS INSTEAD OF DEFINING PATHS IN ARGUMENTS
------------------------------------------------------------------------------------------------------------------------------------

**1. I WANT TO AVOID THIS WARNING THAT COMES FIRST WHILE STARTING THE FORGE**
>>>J:\webui_forge_cu121_torch231\system\python\lib\site-packages\transformers\utils\hub.py:127: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.
  warnings.warn(<<<
===============================================================================

**2. HOW TO RECTIFY THIS VARIABLE SETTING**
>>>Environment vars changed: {'stream': False, 'inference_memory': 1024.0, 'pin_shared_memory': False}<<
================================================================================

**3. WHENEVER I CHANGE THE FLUX MODELS, WHY FORGE CHECKS ALL THE EMBEDDINGS AS BELOW**
ac_neg1.png: no embedded information found.
AS-YoungV2-neg.png: no embedded information found.
BadDream.png: no embedded information found.
bad_prompt_version2-neg.png: no embedded information found.
comicbookpencils.png: no embedded information found.
easynegative.png: no embedded information found.
FastNegativeV2.png: no embedded information found.
ImgFixerPre0.3.png: no embedded information found.
NEG-fixl-2.png: no embedded information found.
negativeXL_D.png: no embedded information found.
negative_hand-neg.png: no embedded information found.
ng_deepnegative_v1_75t.png: no embedded information found.
unaestheticXL_bp5.png: no embedded information found.
verybadimagenegative_v1.3.png: no embedded information found
===================================================================================
👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇👇
**NOW THE COMMAND PROMPTS FROM STARTING TO GENERATION (HIGHLIGHTED THE ERRORS SAID ABOVE)**
--------------------------------------------------------------------------------------------------
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: f2.0.1v1.10.1-previous-304-g394da019
Commit hash: 394da01959ae09acca361dc2be0e559ca26829d4
CUDA 12.1
Launching Web UI with arguments: --precision full --opt-split-attention --always-batch-cond-uncond --no-half --skip-torch-cuda-test --cuda-malloc --cuda-stream --theme dark
Using cudaMallocAsync backend.
Total VRAM 8192 MB, total RAM 65447 MB
pytorch version: 2.3.1+cu121
xformers version: 0.0.27
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 2060 SUPER : cudaMallocAsync
VAE dtype preferences: [torch.float32] -> torch.float32
CUDA Using Stream: True
**_J:\webui_forge_cu121_torch231\system\python\lib\site-packages\transformers\utils\hub.py:127: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.
  warnings.warn(_**
Using xformers cross attention
Using xformers attention for VAE
ControlNet preprocessor location: J:\webui_forge_cu121_torch231\webui\models\ControlNetPreprocessor
22:43:41 - ReActor - STATUS - Running v0.7.1-a1 on Device: CUDA
2024-08-17 22:43:50,873 - ControlNet - INFO - ControlNet UI callback registered.
Model selected: {'checkpoint_info': {'filename': 'J:\\webui_forge_cu121_torch231\\webui\\models\\Stable-diffusion\\Flux\\FLUX.1-schnell-dev-merged-fp8-4step.safetensors', 'hash': '9e0fb423'}, 'additional_modules': [], 'unet_storage_dtype': 'nf4'}
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 28.7s (prepare environment: 4.5s, launcher: 2.5s, import torch: 4.7s, initialize shared: 0.2s, other imports: 1.1s, list SD models: 0.4s, load scripts: 3.3s, initialize extra networks: 7.6s, create ui: 2.9s, gradio launch: 1.5s).
**_Environment vars changed: {'stream': False, 'inference_memory': 1024.0, 'pin_shared_memory': False}_**
Model selected: {'checkpoint_info': {'filename': 'J:\\webui_forge_cu121_torch231\\webui\\models\\Stable-diffusion\\Flux\\flux1DevSchnellBNB_flux1DevBNBNF4.safetensors', 'hash': '0184473b'}, 'additional_modules': [], 'unet_storage_dtype': 'nf4'}
Loading Model: {'checkpoint_info': {'filename': 'J:\\webui_forge_cu121_torch231\\webui\\models\\Stable-diffusion\\Flux\\flux1DevSchnellBNB_flux1DevBNBNF4.safetensors', 'hash': '0184473b'}, 'additional_modules': [], 'unet_storage_dtype': 'nf4'}
[Unload] Trying to free 953674316406250018963456.00 MB for cuda:0 with 0 models keep loaded ...
StateDict Keys: {'transformer': 2350, 'vae': 244, 'text_encoder': 198, 'text_encoder_2': 220, 'ignore': 0}
Using Detected T5 Data Type: torch.float8_e4m3fn
Working with z of shape (1, 16, 32, 32) = 16384 dimensions.
K-Model Created: {'storage_dtype': 'nf4', 'computation_dtype': torch.float16}

**ac_neg1.png: no embedded information found.
AS-YoungV2-neg.png: no embedded information found.
BadDream.png: no embedded information found.
bad_prompt_version2-neg.png: no embedded information found.
comicbookpencils.png: no embedded information found.
easynegative.png: no embedded information found.
FastNegativeV2.png: no embedded information found.
ImgFixerPre0.3.png: no embedded information found.
NEG-fixl-2.png: no embedded information found.
negativeXL_D.png: no embedded information found.
negative_hand-neg.png: no embedded information found.
ng_deepnegative_v1_75t.png: no embedded information found.
unaestheticXL_bp5.png: no embedded information found.
verybadimagenegative_v1.3.png: no embedded information found.**

Model loaded in 9.0s (unload existing model: 0.2s, forge model load: 8.8s).
Skipping unconditional conditioning when CFG = 1. Negative Prompts are ignored.
To load target model JointTextEncoder
Begin to load 1 model
[Unload] Trying to free 7725.00 MB for cuda:0 with 0 models keep loaded ...
[Memory Management] Current Free GPU Memory: 7136.91 MB
[Memory Management] Required Model Memory: 5154.62 MB
[Memory Management] Required Inference Memory: 1024.00 MB
[Memory Management] Estimated Remaining GPU Memory: 958.29 MB
Moving model(s) has taken 5.64 seconds
Distilled CFG Scale: 3.5
To load target model KModel
Begin to load 1 model
[Unload] Trying to free 9411.13 MB for cuda:0 with 0 models keep loaded ...
[Unload] Current free memory is 1159.67 MB ...
[Unload] Unload model JointTextEncoder
[Memory Management] Current Free GPU Memory: 7095.57 MB
[Memory Management] Required Model Memory: 6246.84 MB
[Memory Management] Required Inference Memory: 1024.00 MB
[Memory Management] Estimated Remaining GPU Memory: -175.28 MB
[Memory Management] Loaded to CPU Swap: 1461.51 MB (blocked method)
[Memory Management] Loaded to GPU: 4785.26 MB
Moving model(s) has taken 7.78 seconds
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [01:17<00:00,  3.88s/it]
To load target model IntegratedAutoencoderKL████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [01:12<00:00,  3.82s/it]
Begin to load 1 model
[Unload] Trying to free 8991.55 MB for cuda:0 with 0 models keep loaded ...
[Unload] Current free memory is 2063.04 MB ...
[Unload] Unload model KModel
[Memory Management] Current Free GPU Memory: 6895.08 MB
[Memory Management] Required Model Memory: 319.75 MB
[Memory Management] Required Inference Memory: 1024.00 MB
[Memory Management] Estimated Remaining GPU Memory: 5551.33 MB
Moving model(s) has taken 2.14 seconds
Warning: Ran out of memory when regular VAE decoding, retrying with tiled VAE decoding.
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:05<00:00,  5.44s/it]
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:05<00:00,  5.40s/it]
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:04<00:00,  4.36s/it]
Total progress: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [01:31<00:00,  4.55s/it]
T
----------------------------------------------------------------------------------------------------------
**PLEASE GUIDE TO RECTIFY THESE ERRORS**

**Comment by JollyRoger112** [comment]
Did you find an answer?

**Reply by tmprabubiz** [reply]
NO. I could not find the answer. Following is the one I copied now after starting the Forge and generating an image with flux model
===========================================
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: f2.0.1v1.10.1-previous-531-g210af4f8
Commit hash: 210af4f80406f78a67e1c35a64a6febdf1200a82
CUDA 12.1
Launching Web UI with arguments: --disable-xformers --precision full --opt-split-attention --always-batch-cond-uncond --no-half --skip-torch-cuda-test --cuda-malloc --cuda-stream --theme dark
Using cudaMallocAsync backend.
Total VRAM 8192 MB, total RAM 65451 MB
pytorch version: 2.3.1+cu121
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 2060 SUPER : cudaMallocAsync
VAE dtype preferences: [torch.float32] -> torch.float32
CUDA Using Stream: True
J:\webui_forge_cu121_torch231\system\python\lib\site-packages\transformers\utils\hub.py:127: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.
  warnings.warn(
Using pytorch cross attention
Using pytorch attention for VAE
ControlNet preprocessor location: J:\webui_forge_cu121_torch231\webui\models\ControlNetPreprocessor
22:26:24 - ReActor - STATUS - Running v0.7.1-a2 on Device: CUDA
2024-09-17 22:26:35,339 - ControlNet - INFO - ControlNet UI callback registered.
Model selected: {'checkpoint_info': {'filename': 'J:\\webui_forge_cu121_torch231\\webui\\models\\Stable-diffusion\\Flux\\flux1-dev.safetensors', 'hash': 'b04b3ba1'}, 'additional_modules': ['J:\\webui_forge_cu121_torch231\\webui\\models\\VAE\\ae.safetensors', 'J:\\webui_forge_cu121_torch231\\webui\\models\\text_encoder\\clip_l.safetensors', 'J:\\webui_forge_cu121_torch231\\webui\\models\\text_encoder\\t5xxl_fp16.safetensors'], 'unet_storage_dtype': None}
Using online LoRAs in FP16: False
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 60.6s (prepare environment: 16.6s, import torch: 13.5s, initialize shared: 0.4s, other imports: 1.8s, list SD models: 0.1s, load scripts: 8.3s, initialize extra networks: 9.0s, create ui: 7.3s, gradio launch: 3.4s).
Environment vars changed: {'stream': False, 'inference_memory': 1874.0, 'pin_shared_memory': False}
[GPU Setting] You will use 77.12% GPU memory (6317.00 MB) to load weights, and use 22.88% GPU memory (1874.00 MB) to do matrix computation.
Model selected: {'checkpoint_info': {'filename': 'J:\\webui_forge_cu121_torch231\\webui\\models\\Stable-diffusion\\Flux\\fluxArtfusion4Steps_v12_2.safetensors', 'hash': '24779e58'}, 'additional_modules': ['J:\\webui_forge_cu121_torch231\\webui\\models\\VAE\\ae.safetensors', 'J:\\webui_forge_cu121_torch231\\webui\\models\\text_encoder\\clip_l.safetensors', 'J:\\webui_forge_cu121_torch231\\webui\\models\\text_encoder\\t5xxl_fp16.safetensors'], 'unet_storage_dtype': None}
Using online LoRAs in FP16: False
Loading Model: {'checkpoint_info': {'filename': 'J:\\webui_forge_cu121_torch231\\webui\\models\\Stable-diffusion\\Flux\\fluxArtfusion4Steps_v12_2.safetensors', 'hash': '24779e58'}, 'additional_modules': ['J:\\webui_forge_cu121_torch231\\webui\\models\\VAE\\ae.safetensors', 'J:\\webui_forge_cu121_torch231\\webui\\models\\text_encoder\\clip_l.safetensors', 'J:\\webui_forge_cu121_torch231\\webui\\models\\text_encoder\\t5xxl_fp16.safetensors'], 'unet_storage_dtype': None}
[Unload] Trying to free all memory for cuda:0 with 0 models keep loaded ... Done.
StateDict Keys: {'transformer': 776, 'vae': 244, 'text_encoder': 196, 'text_encoder_2': 220, 'ignore': 0}
Using Default T5 Data Type: torch.float16
Using Detected UNet Type: torch.float8_e4m3fn
Working with z of shape (1, 16, 32, 32) = 16384 dimensions.
K-Model Created: {'storage_dtype': torch.float8_e4m3fn, 'computation_dtype': torch.float16}
ac_neg1.png: no embedded information found.
AS-YoungV2-neg.png: no embedded information found.
BadDream.png: no embedded information found.
bad_prompt_version2-neg.png: no embedded information found.
comicbookpencils.png: no embedded information found.
easynegative.png: no embedded information found.
FastNegativeV2.png: no embedded information found.
ImgFixerPre0.3.png: no embedded information found.
NEG-fixl-2.png: no embedded information found.
negativeXL_D.png: no embedded information found.
negative_hand-neg.png: no embedded information found.
ng_deepnegative_v1_75t.png: no embedded information found.
unaestheticXL_bp5.png: no embedded information found.
verybadimagenegative_v1.3.png: no embedded information found.
Calculating sha256 for J:\webui_forge_cu121_torch231\webui\models\Stable-diffusion\Flux\fluxArtfusion4Steps_v12_2.safetensors: 6168beb7a39c4a49a689d56f5f1ec048836807885c6e7f605310bf7e4bb9cd4b
Model loaded in 10.4s (unload existing model: 0.3s, forge model load: 10.1s).
Skipping unconditional conditioning when CFG = 1. Negative Prompts are ignored.
[Unload] Trying to free 14314.34 MB for cuda:0 with 0 models keep loaded ... Done.
[Memory Management] Target: JointTextEncoder, Free GPU: 7136.91 MB, Model Require: 9569.49 MB, Previously Loaded: 0.00 MB, Inference Require: 1874.00 MB, Remaining: -4306.59 MB, CPU Swap Loaded (blocked method): 5622.00 MB, GPU Loaded: 4019.87 MB
Moving model(s) has taken 4.51 seconds
Distilled CFG Scale will be ignored for Schnell
[Unload] Trying to free 16616.41 MB for cuda:0 with 0 models keep loaded ... Current free memory is 1459.30 MB ... Unload model JointTextEncoder Done.
[Memory Management] Target: KModel, Free GPU: 7095.84 MB, Model Require: 11340.31 MB, Previously Loaded: 0.00 MB, Inference Require: 1874.00 MB, Remaining: -6118.47 MB, CPU Swap Loaded (blocked method): 7344.00 MB, GPU Loaded: 3996.31 MB
Moving model(s) has taken 9.09 seconds
100%|████████████████████████████████████████████████████████████████████████████████████| 4/4 [00:22<00:00,  5.64s/it]
[Unload] Trying to free 6617.87 MB for cuda:0 with 0 models keep loaded ... Current free memory is 2134.55 MB ... Unload model KModel Done.
[Memory Management] Target: IntegratedAutoencoderKL, Free GPU: 7092.41 MB, Model Require: 319.75 MB, Previously Loaded: 0.00 MB, Inference Require: 1874.00 MB, Remaining: 4898.67 MB, All loaded to GPU.
Moving model(s) has taken 2.02 seconds
Total progress: 100%|████████████████████████████████████████████████████████████████████| 4/4 [00:15<00:00,  3.75s/it]
Total progress: 100%|████████████████████████████████████████████████████████████████████| 4/4 [00:15<00:00,  3.09s/it]

**Comment by DenOfEquity** [comment]
1. Transformers library will at some point change the environment variable it uses to identify cache location. You can edit `environment.bat` in the base install directory. The last line is `set TRANSFORMERS_CACHE=...`; you can just delete that line, or if you want to specify the location change it to `set HF_HOME=...`. Forge already sets `HF_HOME` internally.
2. Not an error, or even a warning. Just information relating to how Forge is managing memory.
3. Settings -> Extra Networks -> Print a list of Textual Inversion embeddings when loading model. And/or rename the preview PNGs to `{name}.preview.png`.

edit:
If you've used Spaces, there may be models downloaded to the TRANSFORMERS_CACHE - default location: `{Forge install directory}\system\transformers-cache`. You can simply move the contents of that directory to the HF_HOME location, which by default is `{Forge install directory}\webui\models\diffusers`.

**Reply by tmprabubiz** [reply]
Thank You. Everything worked well. Very happy.
-
I did this - Settings -> Extra Networks -> Print a list of Textual Inversion embeddings when loading model
-
I could not find this - And/or rename the preview PNGs to {name}.preview.png.
-
Please guide

**Reply by DenOfEquity** [reply]
that last one isn't an option in the settings. Open the directory where you've saved your embeddings, and rename the preview images so instead of being `{name}.png` they are `{name}.preview.png`. This is so the loader knows they are preview images, not images with embeddings inside.

**Reply by tmprabubiz** [reply]
Oh..... I don't know how I missed it. Thank You. Please have a look at my command prompt for mistakes
=============================================
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: f2.0.1v1.10.1-previous-535-gb20cb4bf
Commit hash: b20cb4bf0e526f890fcd40a4d039da581cfebafa
CUDA 12.1
Launching Web UI with arguments: --disable-xformers --precision full --opt-split-attention --always-batch-cond-uncond --no-half --skip-torch-cuda-test --cuda-malloc --cuda-stream --theme dark
Using cudaMallocAsync backend.
Total VRAM 8192 MB, total RAM 65451 MB
pytorch version: 2.3.1+cu121
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 2060 SUPER : cudaMallocAsync
VAE dtype preferences: [torch.float32] -> torch.float32
CUDA Using Stream: True
Using pytorch cross attention
Using pytorch attention for VAE
ControlNet preprocessor location: J:\webui_forge_cu121_torch231\webui\models\ControlNetPreprocessor
01:49:39 - ReActor - STATUS - Running v0.7.1-a2 on Device: CUDA
Textual inversion embeddings loaded(21): ac_neg1, AS-YoungV2-neg, Asian-Less2-Neg, bad_prompt_version2-neg, BadDream, BeyondSDXLv3, comicbookpencils, CXL-Home, CXL-Mech, detailxl, easynegative, FastNegativeV2, ImgFixerPre0.3, NEG-fixl-2, negative_hand-neg, negativeXL_D, ng_deepnegative_v1_75t, pureerosface_v1, realisticvision-negative-embedding, unaestheticXL_bp5, verybadimagenegative_v1.3
2024-09-21 01:49:42,274 - ControlNet - INFO - ControlNet UI callback registered.
Model selected: {'checkpoint_info': {'filename': 'J:\\webui_forge_cu121_torch231\\webui\\models\\Stable-diffusion\\Flux\\fluxArtfusion4Steps_v12.safetensors', 'hash': '24779e58'}, 'additional_modules': ['J:\\webui_forge_cu121_torch231\\webui\\models\\VAE\\ae.safetensors', 'J:\\webui_forge_cu121_torch231\\webui\\models\\text_encoder\\clip_l.safetensors', 'J:\\webui_forge_cu121_torch231\\webui\\models\\text_encoder\\t5xxl_fp16.safetensors'], 'unet_storage_dtype': None}
Using online LoRAs in FP16: False
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 30.2s (prepare environment: 7.7s, import torch: 6.8s, initialize shared: 0.1s, other imports: 0.5s, load scripts: 4.1s, initialize extra networks: 0.9s, initialize google blockly: 0.2s, create ui: 6.7s, gradio launch: 3.1s).
Environment vars changed: {'stream': False, 'inference_memory': 1874.0, 'pin_shared_memory': False}
[GPU Setting] You will use 77.12% GPU memory (6317.00 MB) to load weights, and use 22.88% GPU memory (1874.00 MB) to do matrix computation.
Loading Model: {'checkpoint_info': {'filename': 'J:\\webui_forge_cu121_torch231\\webui\\models\\Stable-diffusion\\Flux\\fluxArtfusion4Steps_v12.safetensors', 'hash': '24779e58'}, 'additional_modules': ['J:\\webui_forge_cu121_torch231\\webui\\models\\VAE\\ae.safetensors', 'J:\\webui_forge_cu121_torch231\\webui\\models\\text_encoder\\clip_l.safetensors', 'J:\\webui_forge_cu121_torch231\\webui\\models\\text_encoder\\t5xxl_fp16.safetensors'], 'unet_storage_dtype': None}
[Unload] Trying to free all memory for cuda:0 with 0 models keep loaded ... Done.
StateDict Keys: {'transformer': 776, 'vae': 244, 'text_encoder': 196, 'text_encoder_2': 220, 'ignore': 0}
Using Default T5 Data Type: torch.float16
Using Detected UNet Type: torch.float8_e4m3fn
Working with z of shape (1, 16, 32, 32) = 16384 dimensions.
K-Model Created: {'storage_dtype': torch.float8_e4m3fn, 'computation_dtype': torch.float16}
NEG-fixl-2preview.png: no embedded information found.
Model loaded in 1.7s (unload existing model: 0.3s, forge model load: 1.4s).
Skipping unconditional conditioning when CFG = 1. Negative Prompts are ignored.
[Unload] Trying to free 14314.34 MB for cuda:0 with 0 models keep loaded ... Done.
[Memory Management] Target: JointTextEncoder, Free GPU: 7136.91 MB, Model Require: 9569.49 MB, Previously Loaded: 0.00 MB, Inference Require: 1874.00 MB, Remaining: -4306.59 MB, CPU Swap Loaded (blocked method): 5622.00 MB, GPU Loaded: 4019.87 MB
Moving model(s) has taken 2.60 seconds
Distilled CFG Scale will be ignored for Schnell
[Unload] Trying to free 16616.41 MB for cuda:0 with 0 models keep loaded ... Current free memory is 2573.59 MB ... Unload model JointTextEncoder Done.
[Memory Management] Target: KModel, Free GPU: 7095.57 MB, Model Require: 11340.31 MB, Previously Loaded: 0.00 MB, Inference Require: 1874.00 MB, Remaining: -6118.74 MB, CPU Swap Loaded (blocked method): 7344.00 MB, GPU Loaded: 3996.31 MB
Moving model(s) has taken 3.98 seconds
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 4/4 [00:20<00:00,  5.08s/it]
[Unload] Trying to free 8991.55 MB for cuda:0 with 0 models keep loaded ... Current free memory is 3095.27 MB ... Unload model KModel Done.████████████████████████████████████████████████████████████████████████████████████████████████████| 4/4 [00:13<00:00,  3.69s/it]
[Memory Management] Target: IntegratedAutoencoderKL, Free GPU: 7091.60 MB, Model Require: 319.75 MB, Previously Loaded: 0.00 MB, Inference Require: 1874.00 MB, Remaining: 4897.85 MB, All loaded to GPU.
Moving model(s) has taken 1.84 seconds
Total progress: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 4/4 [00:17<00:00,  4.34s/it]
T
-----------------------------------------------------------------------------------
Why does it say "Using online LoRAs in FP16: False"
